### ISSUE TITLE
Leaking secrets

### ISSUE THREAD

**Issue by dmgerman** [dspinellis/ai-cli-lib#14]
The context option seems to grab N-history lines and send them to the AI. This is potentially a security concern and might be good to document it in the readme. For example, in Postgresql a user might change the password of an account using ALTER user X With password. There might be similar concerns in other programs.

**Comment by dspinellis** [comment]
Agreed that it should be documented. Ideally the program should offer a way to turn-off history recording, as _bash_ does for commands starting with a space. _ai-cli_lib_ could not send commands that start with a space as context.
